src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [filmTitle, searchFilm] = useState('');
+
+  const handleSearch = (event: { target:
+  { value: React.SetStateAction<string>; }; }) => {
+    searchFilm(event.target.value);
+  };
+
+  const [movies] = useState(moviesFromServer);
+
+  const filteredMovies = movies.filter((movie) =>
+    // eslint-disable-next-line implicit-arrow-linebreak
```

Comment:
Avoid disabling ESLint rules unless it is absolutely necessary. In this case, the line break can be avoided by wrapping the return statement of the arrow function in parentheses.